apps/prairielearn/src/schemas/infoQuestion.ts

```
@@ -0,0 +1,228 @@
+      .optional(),
+  })
+  .strict()
+  .describe('Options for externally graded questions.');
+
+export const QuestionJsonSchema = z
+  .object({
+    comment: CommentJsonSchema.optional(),
+    uuid: z
+      .string()
+      .regex(
+        new RegExp('^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$'),
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)